lib/touch-bar-manager.js

```diff
@@ -1,7 +1,8 @@
-const {TouchBar, nativeImage} = require('electron').remote;
+import {remote} from 'electron';
+const {TouchBar, nativeImage} = remote;
```

Comment:
Not sure if the current JS version we're targetting supports this, but have you considered using [deconstructing nested objects](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Nested_object_and_array_destructuring)?

```suggestion
import {remote: {TouchBar, nativeImage}} from 'electron';
```

<sup>Hope I didn't mess up the syntax here</sup>

Reply:
I tried this, but it does not work.

Reply:
Did you check that is not due to my incorrect syntax? Since you're using Babel to compile this code I don't see why we couldn't use a JavaScript version that supports this, right?

Reply:
What is the correct syntax?

Reply:
For me the following works if I run it in my browser or Atom console it works just fine.

```js
let { remote: { TouchBar, nativeImage } } = { remote: { TouchBar: "foo", nativeImage: "bar" } };
console.log(TouchBar, nativeImage);
// ==> "foo bar"
```

Reply:
I think the issue is deconstructing when importing. That is the error I was getting. If you can find a babel plugin that does that we can do it.

Reply:
Yes, it seems that this is not possible for import statements. Alas :disappointed: